src/plots/rectanglePlot.ts

```diff
@@ -276,7 +276,7 @@ export class Rectangle<X, Y> extends XYPlot<X, Y> {
       y: attrToProjector["y"](datum, index, dataset),
       width: attrToProjector["width"](datum, index, dataset),
       height: attrToProjector["height"](datum, index, dataset),
-    };
+    } as SVGRect;
```

Comment:
i think it would be more status quo if you changed the following line to `Pick<DOMRect, "x" | "y" | "width" | "height">` instead

https://github.com/palantir/plottable/blob/2db30e420a54075b8e555aa9db3731b05bfd20a9/src/utils/domUtils.ts#L222